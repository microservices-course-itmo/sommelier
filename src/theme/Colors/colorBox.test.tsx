import ColorBox from './colorBox'
import renderer from 'react-test-renderer'
import React from 'react'

describe('ColorBox component', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<ColorBox color={'#FFFFFF'} colorName={'white'}/>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})