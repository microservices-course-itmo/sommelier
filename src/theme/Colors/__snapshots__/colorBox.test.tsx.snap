// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorBox component matches the snapshot 1`] = `
<div
  className="sc-bdnylx fuJNuL"
  color="#FFFFFF"
  onClick={[Function]}
  onMouseLeave={[Function]}
>
  <span
    className="sc-gtssRu zvDFp"
    color="black"
  >
    white
  </span>
  <span
    className="sc-gtssRu zvDFp"
    color="black"
  >
    #FFFFFF
  </span>
</div>
`;
